aries-site/src/tests/accessibility/axe.js

```diff
@@ -52,17 +52,3 @@ test.before(async t => {
     await t.expect(violations.length === 0).ok(createReport(violations));
   }
 });
-
```

Comment:
Curious why you took this out ? Since resources is no longer a path should we just use another path and still test or ?

Reply:
We don't have any page with that layout anymore since it was only appearing on NavPages with no nav items

Reply:
got it that was the coming soon page!